testutils/BUILD.bazel

```diff
@@ -0,0 +1,10 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+
+go_library(
+    name = "testutils",
+    srcs = [
+        "diff.go",
+    ],
+    importpath = "github.com/bazelbuild/buildtools/testutils",
+    visibility = ["//visibility:public"],
+)
```

Comment:
newline missing at end of file
Run buildifier on it.

Reply:
Done.